packages/cli-poa/src/driverResolver/driver.js

```diff
@@ -0,0 +1,9 @@
+import DriverWrapper from './driverWrapper.js';
+
+export default class Driver {
+  constructor(sessionId, executorUrl) {
+    this.sessionId = sessionId;
+    this.executorUrl = executorUrl;
+    this.helper = new DriverWrapper(this.sessionId, this.executorUrl);
```

Comment:
do we still want to go through helper method or we want to handle it differently now?

Reply:
Creating a direct driver object, instead of relying on wrapper.

Reply:
Done.